AlphaCentauri/Assets/Scripts/RodCasting.cs

```diff
@@ -12,6 +12,10 @@ public class RodCasting : MonoBehaviour
     float verticalPercent = 0f; //up down percent
     bool clicked = false, playHorizontal = false, playVertical = false;
     float amplitude;
+    bool Pulling;
+    public bool isThereABite;
+    public bool IsFishing;
```

Comment:
If you're not accessing it from outside the class you can just set it as private